src/lib/server/migrations.ts

```diff
@@ -436,6 +440,24 @@ const migrations = [
 		run: async () => {
 			await collections.pictures.dropIndex('productId_1').catch(console.error);
 		}
+	},
+	{
+		_id: new ObjectId('669a90d18bc5aaf40c863b63'),
+		name: 'Adding actionSettings nostr to products',
```

Comment:
We should also add it to `runtimeConfig`

Otherwise there will be an error when creating a new product with default action settings - since `ProductForm` since it calls `bind:checked={product.actionSettings.nostr.visible}`